content/docs/command-reference/exp/run.md

```diff
@@ -180,27 +195,36 @@ experiment we just ran (`exp-44136`).
 ## Example: Modify parameters on-the-fly
 
 `dvc exp run--set-param` (`-S`) saves you the need to manually edit the params
-file before running an experiment.
+file before running an experiment. It can override (`foo.bar=value`), append
```

Comment:
Can we use more obvious (from an ML point of view) examples for parameter names instead of `foo.bar`?

Reply:
Tried to use some more meaningful names